x-pack/plugin/kql/src/yamlRestTest/resources/rest-api-spec/test/kql/10_kql_basic_query.yml

```diff
@@ -0,0 +1,262 @@
+setup:
+  - requires:
+      cluster_features: "kql_query_supported"
+      reason: KQL query is not available
+
+  - requires:
+      "test_runner_features": "headers"
+
+  - do:
+      indices.create:
+        index: test-index
+        body:
+          mappings:
+            properties:
+              date_field:
+                type: date
+              text_field:
+                type: text
+              keyword_field:
+                type: keyword
+              integer_field:
+                type: integer
+              double_field:
+                type: double
+
+  - do:
+      headers:
```

Comment:
Is `headers` and `Content-Type` necessary? I don't see them required in other query tests :|

Reply:
You are right. These are useless and I removed them